src/applications/vaos/covid-19-vaccine/components/ContactFacilitiesPage.jsx

```diff
@@ -137,34 +137,27 @@ export default function ContactFacilitiesPage() {
           </li>
         ))}
       </ul>
-      <p className="vads-u-margin-y--3">
-        <NewTabAnchor href="/find-locations">
-          Search for more facilities
-        </NewTabAnchor>
-      </p>
       {!canUseVaccineFlow && (
-        <AlertBox
-          className="vads-u-margin-bottom--1p5"
-          level="2"
-          status="info"
-          backgroundOnly
-          headline="Sign up to stay informed"
-        >
+        <div className="feature">
```

Comment:
What was the reason for changing this from an `AlertBox`?

Reply:
@jbalboni that is what I gathered was the intent of this AC `DESIGN - Blue box uses the featured content component from the design system`

Reply:
Cool, I didn't know that was a distinct pattern from the alert box.